src/components/articles/Article.js

```diff
@@ -1,14 +1,24 @@
 import React from 'react';
-import { Row, Col, Statistic } from 'antd';
-import {IconText} from './ArticleFooter';
+import {
+  Row, Col, Statistic, Icon,
```

Comment:
Is `Icon` from `antd` being used in this file?

Reply:
It is not being used. I will remove it.